### PR TITLE
Corrections to test_aircraft .csv files 

### PR DESCRIPTION
**Pull request by cmbenne3**
### Summary
Updates the variable names and values so that the test aircraft .csv files are correctly processed by the load_inputs function.
When running prob.load_inputs several variables are listed as 'unused' in the verbose DEBUG output.
Changes include update of variable names, units and values.
Changes were tested by running 'prob.load_inputs()' on all 8 .csv files in models/test_aircraft/ in a Jupyter Notebook and a markdown copy of this is attached to the PR
The number of 'unused' variables were reduced however I don't have a translation or correction for the following inputs in these files:

aircraft_for_bench_FwFm.csv
Unused: aircraft:crew_and_payload:num_non_flight_crew [3]
Unused: aircraft:nacelle:count_factor [2]

aircraft_for_bench_FwGm.csv
Unused: aircraft:engine:enable_sizing [False]
Unused: aircraft:crew_and_payload:num_non_flight_crew [3]
Unused: aircraft:nacelle:count_factor [2]

aircraft_for_bench_GwFm.csv
Unused: aircraft:crew_and_payload:num_non_flight_crew [3]
Unused: aircraft:engine:engine_mass_specific [0.21366]
Unused: aircraft:nacelle:count_factor [2]

aircraft_for_bench_solved2dof.csv
Unused: aircraft:crew_and_payload:num_non_flight_crew [3]
Unused: aircraft:nacelle:count_factor [2]

These should either be correctly mapped or removed if not needed.

### Related Issues

- Resolves #

### Backwards incompatibilities

None

### New Dependencies

None
[landing_altitude_test.md](https://github.com/OpenMDAO/Aviary/files/15124994/landing_altitude_test.md)

### CODE REVIEW

File: aviary/models/test_aircraft/aircraft_for_bench_GwGm_solved.csv [hunk 1]
```diff
@@ -151,7 +151,7 @@ mission:summary:fuel_flow_scaler,1,unitless
 mission:takeoff:decision_speed_increment,10,kn
 mission:takeoff:rotation_speed_increment,5,kn
 mission:taxi:duration,0.1677,h
-settings:equations_of_motion,solved
```
Comment (crecine) [inline aviary/models/test_aircraft/aircraft_for_bench_GwGm_solved.csv:154]: I believe this one is supposed to be "solved", @johnjasa can you confirm?

Comment (cmbenne3) [reply aviary/models/test_aircraft/aircraft_for_bench_GwGm_solved.csv:154]: The reason I changed it was I got this error:

image

When I checked enums.py I only saw 3 options and so changed 'solved' to 'solved_2dof' as I guessed they were the same. 
image

Comment (johnjasa) [reply aviary/models/test_aircraft/aircraft_for_bench_GwGm_solved.csv:154]: Yes, `solved` has been changed to `solved_2dof`. That `aircraft_for_bench_GwGm_solved.csv` should just be removed as it's defunct

**Review by crecine — APPROVED** [review]

**Review by johnjasa — APPROVED** [review]
Thanks @cmbenne3, could you please delete the `GwGm_solved.csv` file? This PR is good to go in my view.

**Pull request merged**